services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].module.scss

```diff
@@ -35,3 +35,37 @@
     width: 50%;
   }
 }
+
+.tabs-container {
+  @include breakpoint(md) {
+    margin-left: -$spacing-05;
+  }
+
+  &::before {
+    position: absolute;
+    z-index: 0;
+    left: 0;
+    width: 100vw;
+    height: $spacing-09;
+    border-bottom: 1px solid $layer-active-01;
```

Comment:
Would we want to just use the pixel height here?. For the border-bottom, is there a border token that would make more sense?